llvm/test/CodeGen/PowerPC/aix-cpu-version.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -verify-machineinstrs -mtriple powerpc64-ibm-aix-xcoff  < %s | FileCheck %s
+
+; CHECK:      .file "1.c"
+; CHECK-NEXT: .machine "ppc64"
```

Comment:
Does system `as`'s `ppc64` and LLVM's `ppc64` refer to the same instruction set?